src/main.ts

```diff
@@ -141,11 +136,74 @@ export async function activate(context: ExtensionContext) {
 		traceOutputChannel,
 	};
 
-	client = new LanguageClient(
-		"biome_lsp",
-		"Biome",
-		serverOptions,
-		clientOptions,
+  
+	const reloadClient = async () => {
+		outputChannel.appendLine(`Using Biome from ${server.command}`);
+
+		let tmpDestination: string;
+
+		try {
+			tmpDestination = path.resolve(
+				os.tmpdir(),
+				`./biome${process.platform === "win32" ? ".exe" : ""}`,
+			);
+			outputChannel.appendLine(`Copying file to tmp folder: ${tmpDestination}`);
+			copyFileSync(server.command, tmpDestination);
```

Comment:
I think we should prefer using the `workspace.fs` api to copy the file, if possible.

https://code.visualstudio.com/api/references/vscode-api#FileSystem